utils/package.json

```diff
@@ -7,6 +7,8 @@
   },
   "main": false,
   "exports": {
+    "./contextCompat": "./contextCompat.js",
+    "./contextCompat.js": "./contextCompat.js",
```

Comment:
```suggestion
```
the only reason the other extensioned ones are there was to make it a nonbreaking change; extensions in specifiers are bad :-)

Reply:
Done.  And I agree, I was just following the pattern.